scripts/pin_the_slow_way.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env conda-execute
+
+# conda execute
+# env:
+#  - python
+#  - conda-smithy
+#  - gitpython
+#  - pygithub
+# channels:
+#  - conda-forge
+# run_with: python
+
+import os
+import time
+import argparse
+from contextlib import contextmanager
+import textwrap
+import random
+import re
+
+import git
+import github
+
+import conda_smithy.github
+import conda_smithy.configure_feedstock
+import conda_smithy
+
+import conda_smithy.feedstocks as feedstocks
+
+
+pinned = {
+          'boost': 'boost 1.60.*',
+          'bzip2': 'bzip2 1.0.*',
+          'fontconfig': 'fontconfig 2.11.*',
+          'freetype': 'freetype 2.6*',
+          'hdf5': 'hdf5 1.8.17*',
+          'icu': 'icu 56.*',
+          'jpeg': 'jpeg 9*',
+          'libnetcdf': 'libnetcdf 4.4*',
+          'libpng': 'libpng >=1.6.21,<1.7',
+          'libtiff': 'libtiff 4.0.*',
+          'ncurses': 'ncurses 5.9*',
+          'openssl': 'openssl 1.0*',
+          'readline': 'readline 6.2*',
+          'sqlite': 'sqlite 3.9.*',
+          'tk': 'tk 8.5*',
+          'xz': 'xz 5.0.*',  # To be changed once we have the Python recipe.
+          'zlib': 'zlib 1.2*',
```

Comment:
👍

Though we should refactor this out into maybe a yaml file so that it is more visible, is a bit closer to documentation, and is easier to include in other documentation.